cl/opinion_page/views.py

```diff
@@ -387,8 +368,10 @@ def view_opinion(request, pk, _):
             "favorite_form": favorite_form,
             "get_string": get_string,
             "private": cluster.blocked,
-            "citing_clusters": citing_clusters,
-            "top_authorities": cluster.authorities[:5],
+            "top_children": children[:5],
+            "children_count": len(children),
```

Comment:
Just checking: You tested that `len()` works like this? I'm a bit surprised, but I've been away from the Sunburnt APIs for a bit.

Reply:
Changes reverted in f85e738